alpha/apps/kaltura/lib/storage/urlTokenizers/kCdnVideoUrlTokenizer.php

```diff
@@ -0,0 +1,43 @@
+<?php
+class kCdnVideoUrlTokenizer extends kUrlTokenizer
+{
+	/**
+	 * @param string $url
+	 * @param string $urlPrefix
+	 * @return string
+	 */
+	public function tokenizeSingleUrl($url, $urlPrefix = null)
+	{
+		return $this->tokenizeUrl($url);
+	}
+	/**
+	 * @param $url
+	 * @return string
+	 */
+	private function tokenizeUrl($url)
+	{
+		$expiryTime = time()+$this->getWindow();
+		$baseUrl = parse_url($url);
+		$hashData = $this->getKey().":$expiryTime:$baseUrl[path]";
+		$token = $this->getTokenByData($hashData);
+
+		if (strpos($url, '?') !== false)
```

Comment:
@TalDubovKaltura please make sure to use curly brackets when opening if-else blocks.

Reply:
@hilak I tried to keep the uniformity here, it's the same code snippet you're using in ChinaCache and also another two tokenizers(ChtHttp and BitGravity). If you think so I can change all of them.